Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -635,12 +635,19 @@ GDCMImageIO::InternalReadImageInformation()
   switch (ms)
   {
     case gdcm::MediaStorage::HardcopyGrayscaleImageStorage:
+      [[fallthrough]];
     case gdcm::MediaStorage::GEPrivate3DModelStorage:
+      [[fallthrough]];
     case gdcm::MediaStorage::Philips3D:
+      [[fallthrough]];
```

Comment:
@hjmjohnson Just wondering, did the original code cause any warning?

Reply:
The C++ core guideline is to use `[[fallthrough]]` only for non-empty cases: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es78-dont-rely-on-implicit-fallthrough-in-switch-statements

It says:

> Multiple case labels of a single statement is OK:

```cpp
switch (x) {
case 'a':
case 'b':
case 'f':
    do_something(x);
    break;
}
```

Reply:
Yeah, I wouldn't put any `[[fallthrough]]` on empty cases either.

I don't think the old way would have had any warnings, at least not from clang's `-Wimplicit-fallthrough`.

I just noticed I don't have that flag enabled on my bots... I will check if there are none and enable it...